tags/resources/toc-supporting-guides/tech-papers.md

```diff
@@ -0,0 +1,155 @@
+
+### Refining the scope, audience and participants
+
+Interest in participating is expressed on the Issue by comments of those individuals. Soliciting interest and participation must allow for sufficient time (no less than two weeks) for fair opportunity to gather interest. Interested parties should then meet to define timelines, refine the scope, declare the audience, and determine paper lead. It is recommended that TAGs leverage communication channels to increase awareness of the proposed paper, its topic, and to solict potential contributors.
+
+## Managing paper development
+
+Once the group has met to define timelines, scope, audience, and determined the lead, the Issue must be updated to provide the following information (initial estimates provided but may be adjusted by the group or TAG leadership as appropriate):
+
+| Milestone | Estimated Time |
+| --- | --- |
+| Audience, Goals, & refining scope | 1 week |
+| Outline | 2 weeks |
+| Tasking Assignment | 1 week |
+| Content Rough-in | 2-3 weeks |
+| Collaborative Review | 2 weeks |
+| Executive Summary and content wrap up | 2 weeks |
+| Narrative Voice | 1-2 weeks |
+| Final Group Review | 1 week |
+| Community Review | 2 weeks |
+| Public comment adjudication | 2 weeks (simultaneous with review) |
+| Leadership Review | 1-2 weeks |
+| CNCF publishing engagement | ~2-3 weeks |
+| Addition to the repo | 2 weeks |
+| Blog post and publishing coordination | 2-3 weeks |
+
+### Milestones Details
+
+#### Audience, Goals, and Refining Scope
+
+Define the following:
+
+- Who is the intended audience?
+- What positions do they hold?
+- What problem are we addressing?
+- What assumptions are we making about the audience or content?
+- Document these in the "Introduction > Assumptions" section.
+
+#### Outline
+
+The outline needs crafted and placed in a shared document (google docs, hackmd, or other collaborative documentation tool). It will serve as the basis for paper structure and tasking. The Issue must be updated to link to the shared document.
+
+#### Tasking Assignment
+
+Contributors should assign themselves to various outline areas. This allows focus and identifies gaps needing more contributors. Use tools like Google Docs' "assign to me" feature for task assignment. Alternatively, participants may tag themselves in line by typing their name, or other agreed upon option to clarify who is responsible for which portions of the paper.
+
+#### Content Rough-in
+
+Assignees provide rough content through phrases, paragraphs, or bullets. If the assignee has drafted complete content (full sentences, ideas, well-expressed) they should indicate the content as "draft". This informs other collaborators that less effort may be needed to further develop the content and may facillitate comments to refine the material further or restructure as content is further developed.
```

Comment:
The beginning of this section initially confused me. A general overview of the various stages a whitepaper needs to go through (e.g. scope discussion, rough-in, draft, public comments, publication) could help clarify.

Reply:
added above.